frontend/src/libs/components/app/app.tsx

```diff
@@ -16,6 +33,14 @@ const App: React.FC = () => {
     }
   }, [isRoot, dispatch]);
 
+  useEffect(() => {
+    void dispatch(authActions.getAuthenticatedUser());
+  }, [dispatch]);
+
+  if (authenticatedUserDataStatus !== DataStatus.FULFILLED) {
```

Comment:
@Dmytro-Kucherenko I like the comment you wrote, so keep going this way) Functional components works such a way that whenever your component re-renders the whole function (your component) will be executed again, that's why we need hooks that keep data despite re-renders. So this will work the same way)

Reply:
Okay, thanks for the clarification)